cli/ts/commands/proveOnChain.ts

```diff
@@ -48,6 +49,7 @@ import { Proof } from "../utils/interfaces";
 export const proveOnChain = async (
   pollId: string,
   proofDir: string,
+  deploySubsidy: boolean,
```

Comment:
maybe let's give it a more appropriate name, `subsidyEnabled` maybe?

Reply:
that sounds great, I just randomly put a name